extensions/all-extensions/pom.xml

```diff
@@ -29,6 +29,22 @@
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-validation</artifactId>
             <version>${project.version}</version>
+            <type>zip</type>
+            <!--TODO(gfernandes): dependencies included in modules as zip must be excluded as this artifact
```

Comment:
This is not a TODO, move it before the dependencies section begins, so every one adding a new dependency will be aware of it